### ISSUE TITLE
Slow compile times

### ISSUE THREAD

**Issue by raniejade** [raniejade/godot-kotlin#31]
Yes, it is very slow. https://github.com/JetBrains/kotlin-native/issues/733

**Comment by Erik262** [comment]
Yeah same here. Sitting in front of an i9900K CPU beast. Compiling time for a simple "Hello World" output needs about 4 seconds...
